panel/__init__.py

```diff
@@ -65,3 +65,38 @@
 from .param import Param # noqa
 from .template import Template # noqa
 from .widgets import indicators # noqa
+
+__all__ = [
+    "__version__",
+    "Accordion",
+    "bind",
+    "Card",
+    "Column",
+    "config",
+    "depends",
+    "extension",
+    "FlexBox",
+    "GridBox",
+    "GridSpec",
+    "indicators",
+    "interact",
+    "ipywidget",
+    "layout",
+    "links",
+    "pane",
+    "Pane",
+    "panel",
+    # "param",
```

Comment:
I don't think `pn.param` is to be exposed?

Reply:
Seems like users will sometimes need to instantiate the Param pane, ParamMethod, and ParamFunction, so I don't see why not...

Reply:
If those funktions are considered public they should be documented.

I have never read about them and would not know what they do or when to use?

Reply:
You use them **all the time**. They are what generates widgets from Parameters and what makes reactive functions (i.e. those wrapped with depends and bind) interactive.